app/src/main/java/com/brainwallet/ui/screens/home/composable/HomeSettingDrawerSheet.kt

```diff
@@ -57,7 +58,10 @@ fun HomeSettingDrawerSheet(
     val context = LocalContext.current
 
     LaunchedEffect(Unit) {
-        viewModel.onEvent(SettingsEvent.OnLoad(BRSharedPrefs.getShareData(context))) //currently just load analytics share data here
+        viewModel.onEvent(SettingsEvent.OnLoad(
```

Comment:
@andhikayuana thanks for looking at this. The idea is that that when a sync completes the settings automatically updates showing the metadata